testing_view.py

```diff
@@ -33,3 +33,23 @@ def testing(request):
     }
     return HttpResponse(template.render(context, request))
 https://github.com/samson-olofinsae/Dj-pro.git
+
+#  view for numeric if in template  (greeting ==1) added  below
+
+def testing (request):
+    template = loader.get_template ('template.html')
+    context = {
+    'greeting' : 1,
+}
+
+    return HttpResponse (template.render (context, request))
+
+# view for IF/ELSE  tag
+#def testing (request):
```

Comment:
how about identation?

Reply:
indentation not neccesary  for commented-out session